lsp-protocol.el

```diff
@@ -153,16 +153,20 @@ See `-let' for a description of the destructuring mechanism."
 
 ;; manually defined interfaces
 (lsp-interface (RpcRequestResponse (:params :id :method) nil))
+(lsp-interface (Command (:title :command) (:arguments)))
 
 (lsp-interface (css:ApplyCodeAction (:arguments) nil))
 
-(lsp-interface (eslint:StatusNotification (:state) nil))
-(lsp-interface (eslint:WorkspaceConfigurationReqHandler (:items) nil))
-(lsp-interface (eslint:WorkspaceConfiguration (:scopeUri) nil))
-(lsp-interface (eslint:OpenDoc (:url) nil))
+(lsp-interface (eslint:StatusNotification (:state) nil)
+               (eslint:WorkspaceConfigurationReqHandler (:items) nil)
+               (eslint:WorkspaceConfiguration (:scopeUri) nil)
+               (eslint:OpenDoc (:url) nil))
 
 (lsp-interface (haxe:ProcessStartNotification (:title) nil))
 
+(lsp-interface (pwsh:ApplyCodeActionEdits (:StartLineNumber :EndLineNumber :StartColumnNumber :EndColumnNumber :Text) nil)
+               (pwsh:ApplyCodeActionEditsRequest (:range :newText) nil))
```

Comment:
What's this used for?

Reply:
It'll be used with `lsp-make-*` when the `plists` branch got merged. I'll remove until the merge.

Reply:
In case you haven't notice, it's `TextEdit` type